values.yaml

```diff
@@ -20,7 +22,7 @@ image:
   # See: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   #pullSecret: "your-pull-secret"
 
-
+clusterDomain: cluster.local
```

Comment:
This needs some kind of doc string.

The clusterDomain is defined here: https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration
And the DNS server has to be configured with it as well:
https://stackoverflow.com/a/52941407/1134951

Looks like kube-dns is the thing that defaults to `cluster.local` (overridable with the `--domain` param): https://github.com/kubernetes/dns/blob/master/cmd/kube-dns/app/options/options.go#L59

I use coredns where there is not a default domain, and I did not use the `cluster.local` domain shown in the docs: https://coredns.io/plugins/kubernetes/

How is your cluster's dns configured?

Reply:
I use coredns.
My configmap:
```.:53 {
    errors
    health
    kubernetes cluster.local in-addr.arpa ip6.arpa {
      pods insecure
      fallthrough in-addr.arpa ip6.arpa
    }
    prometheus :9153
    forward . /etc/resolv.conf
    cache 30
    loop
    reload
    loadbalance
}
```


```

Reply:
on the node my kubelet-config.json does have a line for:
````
{
...
    clusterDomain": "cluster.local",
...
}
```

Reply:
I added a comment here as well.